src/mailDialog.cpp

```diff
@@ -0,0 +1,28 @@
+//
+// Created by Leonard Geier on 15.06.18.
+//
+
+#include <iostream>
+#include "mailDialog.hpp"
+
+MailDialog::MailDialog()
+        : QDialog()
+        , m_title("<font size=\"6\">Bitte gib deine E-Mail an:</font>")
+        , m_buttonOk("OK")
+        , m_buttonCancel("Cancel")
+        , m_input()
+        , m_layout() {
+
+    QObject::connect(&m_buttonOk, &QPushButton::clicked, [=](){
+        emit enteredMail(this->m_input.text());
```

Comment:
For user experience: First close the mail window and then send the mail?
`    QObject::connect(&m_buttonOk, &QPushButton::clicked, [=](){  
	   this->close();  
           emit enteredMail(this->m_input.text());  
    });`